package.json

```diff
@@ -4,43 +4,66 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
+  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
   "license": "MIT",
   "scripts": {
-    "build": "npm run lint && tsc",
-    "test": "jest",
+    "build": "tslint --project . && tsc",
+    "clean": "rm -r ./lib",
+    "prettify": "prettier --write './**/*.ts'",
+    "start": "node lib/index.js",
+    "test": "jest -i",
+    "test:integration": "jest -i --config=jest.integration.js",
     "lint": "tslint --project ."
   },
   "devDependencies": {
-    "prettier": "^1.12.1",
-    "danger": "^3.7.11",
-    "tslint": "^5.10.0",
-    "typescript": "^2.8.3",
+    "@types/bluebird": "^3.5.20",
```

Comment:
it looks like we have typings for bluebird but the library is not installed. why we need this ?